{
  "uid" : "7e461ef57ed4a546",
  "name" : "test_broken",
  "fullName" : "test_allure#test_broken",
  "historyId" : "985c3d972d231f87d0914b89f78754fd",
  "time" : {
    "start" : 1606189429217,
    "stop" : 1606189429217,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "Exception: oops",
  "statusTrace" : "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_allure.py:18: Exception",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "test_allure"
  }, {
    "name" : "host",
    "value" : "USER-8VMLUTRGV1"
  }, {
    "name" : "thread",
    "value" : "15128-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test_allure"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "7e461ef57ed4a546.json",
  "parameterValues" : [ ]
}